modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryServiceTest.java

```diff
@@ -563,6 +679,42 @@ public void testGetObjectEntryHierarchy() throws Exception {
 			objectEntry -> Assert.assertNotNull(
 				_objectEntryService.getObjectEntry(
 					objectEntry.getObjectEntryId())));
+
+		// User can view bound entries if allowed to update root
```

Comment:
What do you think about this comment?
//User can view an object entry _of_ a descendant object definition with the update permission.